cmd/skopeo/main.go

```diff
@@ -54,6 +56,12 @@ func createApp() *cli.App {
 		if c.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+		if c.GlobalIsSet("cert-path") {
+			logrus.Warn("'--cert-path' is deprecated, please set this on the specific subcommand")
+		}
+		if c.GlobalIsSet("tls-verify") {
+			logrus.Warn("'--tls-verify' is deprecated, please set this on the specific subcommand")
```

Comment:
I believe we can just Warn with `--tls-verify` because we still need to support that - I'll totally drop `--cert-path` though since hardly anyone is using that and there's no point in keeping backward compatibility.